CHANGELOG.md

```diff
@@ -35,6 +35,9 @@ Line wrap the file at 100 chars.                                              Th
   default 1500. The 1500 is still the fallback if this for some reason fails. This may stop
   fragmentation.
 
+#### Linux
+- GUI: Add electron flags to run Wayland native if in a Wayland session.
```

Comment:
Please add something along the lines of "in Window managers known to work well".